src/H5Tpkg.h

```diff
@@ -291,6 +297,19 @@ typedef struct H5T_array_t {
     size_t   dim[H5S_MAX_RANK]; /* size in each dimension       */
 } H5T_array_t;
 
+/* Form (Rectangular, Polar, Exponential) of complex number */
+typedef enum H5T_complex_form_t {
+    H5T_COMPLEX_RECTANGULAR,
+    H5T_COMPLEX_POLAR,
+    H5T_COMPLEX_EXPONENTIAL,
```

Comment:
Some compilers are grumpy about the last item in an enum having a ','

Reply:
It's allowed in C99 though! Sounds like some compilers need to be fixed.

Reply:
🤷‍♂️